modules/parachains/src/weights.rs

```diff
@@ -55,49 +55,17 @@ pub trait WeightInfo {
 	fn submit_parachain_heads_with_16kb_proof() -> Weight;
 }
 
-/// Weights for `pallet_bridge_parachains` using the Millau node and recommended hardware.
-pub struct BridgeWeight<T>(PhantomData<T>);
-impl<T: frame_system::Config> WeightInfo for BridgeWeight<T> {
-	fn submit_parachain_heads_with_n_parachains(p: u32) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			.saturating_add(Weight::from_ref_time(18_706_000 as u64).saturating_mul(p as u64))
-			.saturating_add(T::DbWeight::get().reads(2 as u64))
-			.saturating_add(T::DbWeight::get().reads(2 as u64).saturating_mul(p as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64).saturating_mul(p as u64))
-	}
-
-	fn submit_parachain_heads_with_1kb_proof() -> Weight {
-		Weight::from_ref_time(27_549_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(4 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
-	}
-
-	fn submit_parachain_heads_with_16kb_proof() -> Weight {
-		Weight::from_ref_time(80_792_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(4 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
-	}
-}
-
 // For backwards compatibility and tests
 impl WeightInfo for () {
```

Comment:
And this https://github.com/darwinia-network/darwinia/blob/main/runtime/pangoro/src/pallets/bridge_parachains.rs#L35